File: RestApi/HttpClient/Communication/HttpMatchResponse.cs
using BasketballClient.Models;
using System.Net;

namespace BasketballClient.Communication
{
    public class HttpMatchResponse
    {
        public Match Match { get; set; }

        public bool IsSuccess { get; set; }

        public HttpStatusCode StatusCode { get; set; }

        public string ErrorMessage { get; set; }
    }
}

File: RestApi/JavaHttpClient/src/main/java/http/HttpMatchResponse.java
package http;

import models.Match;

public class HttpMatchResponse {
    private Match match;

    private Boolean isSuccess;

    private Integer statusCode;

    private String errorMessage;

    public HttpMatchResponse() {}

    public Match getMatch() {
        return match;
    }

    public void setMatch(Match match) {
        this.match = match;
    }

    public Boolean getSuccess() {
        return isSuccess;
    }

    public void setSuccess(Boolean success) {
        isSuccess = success;
    }

    public Integer getStatusCode() {
        return statusCode;
    }

    public void setStatusCode(Integer statusCode) {
        this.statusCode = statusCode;
    }

    public String getErrorMessage() {
        return errorMessage;
    }

    public void setErrorMessage(String errorMessage) {
        this.errorMessage = errorMessage;
    }
}
